lib/repl.js

```diff
@@ -415,9 +416,11 @@ function REPLServer(prompt,
   });
 
   var sawSIGINT = false;
+  var sawCtrlD = false;
```

Comment:
@thefourtheye `sawCtrlD` is set to `true` after `turnOffEditorMode`. we can execute code only when the editor mode is off